app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -159,4 +159,38 @@ public function processInvitationcall($observer)
         }
     }
 
+    /**
+     * @throws Exception
+     */
```

Comment:
What's the purpose of adding these `@throws` comments? I don't think they're useful.